lib/sycamore/sycamore/connectors/duckdb/duckdb_writer.py

```diff
@@ -104,9 +107,10 @@ def create_target_idempotent(self, target_params: BaseDBWriter.TargetParams):
         client = duckdb.connect(str(dict_params.get("db_url")))
         try:
             if schema:
+                embedding_size = schema.get("embeddings").str.split("[")[0] + str(dict_params.get("dimensions")) + "]"
```

Comment:
does this capture the [ or are you gonna end up with `FLOAT768]`?

Reply:
Changed it